docs/user-guide/creating-application/deployment-template.md

```diff
@@ -173,6 +173,15 @@ image:
 
 Image is used to access images in kubernetes, pullpolicy is used to define the instances calling the image, here the image is pulled when the image is not present,it can also be set as "Always".
 
+### imagePullSecrets
+
+`imagePullSecrets` contains the docker credentials that are used for accessing a registry. 
+
+```yaml
+imagePullSecrets:
+  - docker-credentials
```

Comment:
this will take secret's name where docker credentials exist, in many documents it is `regcred` so let's use regcred instead of docker-credentials.

Reply:
Do mention that regcred is a secret which should exist in the namespace